#167618839 Fix Bugs in Database Config Path

chialuka opened the pull request:
#### What does this PR do?
- Fixes the bug in the database config path for seeds

#### Description of Task proposed in this pull request?
- The `.sequelizerc` file changed. A correct path was added to enable seeding the database

- The `package.json` scripts were updated to show these changes

- Migrations were also run and models and seeds generated to properly test the work. This brought about changes in the four files in the database folder

#### How should this be manually tested (Quality Assurance)?
- Clone the repo, run `npm install`, and then run `npm test`

#### What are the relevant pivotal tracker stories?
- [#167618839](https://www.pivotaltracker.com/story/show/167618839)

#### Any background context you want to add (Operations Impact)?
- Run `npm install` and then run `npm db:rollback`

#### What I have learned working on this feature?
- I have learned how to properly specify migrations for sequelize

ayodejiAA reviewed (dismissed):
Good

dinobi reviewed (changes requested):
Remove the Schema files and implement my feedback for those

chialuka commented:
> Remove the Schema files and implement my feedback for those

I have implemented the feedback on the migrations file since that is where Sequelize effects those changes and not the models. I made email and username unique and made some fields non-nullable.

I had to add the migrations because `npm db:rollback` runs with the test script and if there are no migrations then it will fail on Travis.

I'll remove them right away but that will mean that with another PR there will be a change to the `package.json` file to add the migrations to the test script.

chialuka committed:
bug-[167618839]: fix bugs with database config paths
- Fix bug in .sequelizerc seeders path
- Remove coverage pre-commit hook
[Delivers #167618839]
